src/js/helpers/helpHelper.js

```diff
@@ -137,6 +156,35 @@ export const loadHelp = () => {
 
 }
 
+export const loadTechnical = () => {
+
+	const deferred = Q.defer();
+
+	const output = {
+		html: '',
+		sections: [],
+		history: ''
+	};
+
+	loadTechnicalNotes()
+		.then((data) => {
+			output.html = data.html;
+			output.sections = data.sections;
+			return loadHistory();
+		})
+		.then((data) => {
+			output.history = data;
```

Comment:
Is this actually accessed anywhere? If not, do we even need this section? It seems like it wouldn't be since you only load history on the `/history` page, which is a new call to the md files and just loads `history.md`. `technicalHistory.md` isn't referenced anywhere at all.